src/abi/prosopo.json

```diff
@@ -1,14 +1,14 @@
 {
   "source": {
-    "hash": "0xad56754944fca5da205a2dbcb86b0d674e58df770bf75b862601e2812d301f7a",
+    "hash": "0xb398c34eeaa6631344073ab704806495f35c84bc73f79e69784e4dbf5ad96b02",
     "language": "ink! 4.0.0-beta",
-    "compiler": "rustc 1.68.0-nightly",
+    "compiler": "rustc 1.67.0",
```

Comment:
are we downgrading rust?

Reply:
Not intentionally, no. We need to agree a procedure for applying the most recent contract artifacts to this repo.